src/collections/blog/2022-01-07-Recap-KubeCon-China-TAG-Network-talk/index.mdx

```diff
@@ -0,0 +1,121 @@
+</p>
+
+<p>
+    Lee Calcote (<a href="https://twitter.com/lcalcote">@lcalcote</a>) is an innovative product and technology leader, passionate about empowering engineers and enabling organizations. As the founder and CEO of Layer5, he is at the forefront of the cloud native movement. Ken Owens (<a href ="https://twitter.com/kenowens12">@kenowens12</a>) is the Vice President, Cyber Cloud Security Engineering at FISERV. Owens is a technologist and an innovator with more than 20+ years of experience in a plethora of technologies and has been championing the cloud native movement for more than a decade now. 
+</p>
+<p>
+    As an introduction to TAG Network, Owens  describes  the group’s  core objectives and reasons for its formation, summarized below:
+</p>
+<ul>
+<li>Listening to ideas and collaborations</li>
+<li>Assisting and attracting projects</li>
+<li>Building communities and providing impartial stewardship</li>
+</ul>
+<p>
+    Calcote and Owens are frequently asked a common question <em>“Is it important for the organization to be a part of CNCF to participate in the TAG Network group?”</em>. Owens emphasizes that anyone can participate, regardless of whether or not their company is a CNCF member organization. It does not require any payment and is an open-source effort in its spirit.
+</p>
+<h3>CNCF TAG Network Projects and various stages they go through:</h3>
+<img src={Projects} className="slides-right" align="center" alt="Tag Networks Projects" />
+
+<p>
+    One of the important goals of the TAG Network is to identify the unique projects that want to participate and shepherd their journey through CNCF project levels. Project-fit and the process of assessment for inclusion in the CNCF require your project to be on terms with the following essential traits:
+</p>
+<ul>
+<li>Uniqueness</li>
+<li>Problem it solves</li>
+<li>Value it brings to the community</li> 
+<li>Market demand</li>
+<li>Diversification </li>
+</ul>
+<p>
+    Creating and stewarding any one of these projects is a significant undertaking. The  project's maintainers pitch the project and explicate how their project meets the above traits. 
+</p>
+<p>
+    You start in a sandbox and work your way up to an incubation area once you've demonstrated a fundamental interest and understanding of the space, as well as some adoption. To graduate, the project must satisfy the required traits, be open-source, provide value and solve a problem effectively. We don't want to promote a CNCF project that is solely driven by one source. Diversification is crucial to the core of any CNCF project. Owens shared a subtle analogy, <em>"a truly open source project is one in which contributors belong to the Companies A+B+C...Z instead of just company A+A+A..."</em> 
```

Comment:
```suggestion
    You start in a sandbox and work your way up to an incubation area once you've demonstrated a fundamental interest and understanding of the space, as well as some adoption. To graduate, the project must satisfy the required traits, be open-source, provide value and solve a problem effectively. We don't want to promote a CNCF project that is solely driven by one source. Diversification is crucial to the core of any CNCF project. Owens shared a subtle analogy, <em>"a truly open source project is one in which contributors belong to the companies A+B+C...Z instead of just company A"</em> 
```